.github/workflows/basic.yml

```diff
@@ -158,5 +158,15 @@ jobs:
         python tests/run_spark_hdfs.py $GENOMICSDB_BUILD_DIR $CMAKE_INSTALL_PREFIX local hdfs://localhost:9000/ client $GENOMICSDB_RELEASE_VERSION $GITHUB_WORKSPACE/tests Coverage
         $GITHUB_WORKSPACE/.github/scripts/test_hdfs_htslib_support.sh
 
+    - name: Coverage
+      shell: bash
+      working-directory: ${{env.GENOMICSDB_BUILD_DIR}}
+      run: |
+        lcov --directory . --capture --output-file coverage.info
```

Comment:
After looking at these changes and your question about spark stuff...I wonder if we need to add multiple `--directory` arguments here? It seems we write to `target/jacoco-reports/...` for the CI tests.

Not sure if `--directory` means we search all subdirectories or not...

Reply:
Yeah, `--directory` means search all subdirectories, but `lcov` only looks for `gcda` files. `jacoco` does it similarly for java tests. I see this from my build directory with coverage for the basic test. Will try out `run_spark_hdfs.py` and see what I find. 
```
~/GenomicsDB/build_cov: find . -name jaco*xml -exec ls -l {} \;
-rw-r--r--  1 nalini  staff  1616007 Mar 22 20:36 ./target/jacoco-reports/jacoco-ut/jacoco.xml
-rw-r--r--@ 1 nalini  staff  14555514 Mar 22 21:44 ./target/jacoco-reports/jacoco-ci/jacoco-ci.xml
```